src/pages/patientView/genomicOverview/GenomicOverview.tsx

```diff
@@ -14,26 +15,41 @@ interface IGenomicOverviewProps {
     sampleColors: {[s:string]:string};
     sampleManager: SampleManager;
 }
-export default class GenomicOverview extends React.Component<IGenomicOverviewProps, {vafPlotData:{[s:string]:number[]}}> {
+export default class GenomicOverview extends React.Component<IGenomicOverviewProps, { vafPlotData:{[s:string]:number[]}}> {
```

Comment:
while we're here, can we make the type a bit more specific `vafPlotData:{[sampleId:string]:number[]}`